DTS-STN/Service-Canada-Labs
# Virtual Assistant Update/Blog Page

# Description

Created a page for VA update posts that will be generated based on markdown content from AEM. Created content model for VA update posts which will all be fetched at build time and generated bilingually like all other pages.

The AEM Folder for VA project update **pages** can be found [here](https://author-canada-prod.adobecqms.net/assets.html/content/dam/decd-endc/content-fragments/sclabs/projects/virtual-assistant/project-update-pages)

The AEM Folder for creating project update **cards** on the VA overview page that will link to these posts can be found [here](https://author-canada-prod.adobecqms.net/assets.html/content/dam/decd-endc/content-fragments/sclabs/projects/virtual-assistant/project-updates)

The url used in the card must reflect the url of the blog post being created, in both languages.

The content model is a copy of the SCLabsPagev1 model without additional fields.



## Acceptance Criteria

Update posts created in AEM should be generated as update/blog posts on the site.

## Test Instructions

1. yarn dev or build + start
2. Navigate to Virtual Assistant page
3. Click on link to update/blog post
4. See that page renders properly

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
